src/screens/About/About.style.tsx

```diff
@@ -48,3 +46,37 @@ export const PartnersSectionWrapper = styled.div`
 export const StyledGridContainer = styled(Grid)`
   margin: 0.5rem 0 1.25rem;
 `;
+
+export const CommitmentItem = styled.div`
+  display: flex;
+  align-items: center;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  justify-content: flex-start;
+`;
+
+export const CommitmentIcon = styled.img`
+  margin: 1rem;
+`;
+
+// needed to override end margin for markdown text.
+export const CommitmentBody = styled.div`
+  p {
+    margin-block-end: 0;
+  }
+`;
+
+export const SectionContent = styled.div`
```

Comment:
SectionContent was introduced to help get consistent margin-bottom among different kinds of things in sections (non-Markdown vs Markdown, mainly)

Reply:
thoughts on making a styled component off of MarkdownContent (ie. `StyledMarkdownContent = styled(MarkdownContent)`....``) and setting margins on p to `margin:0` and using `gap` instead (totally new to me since seeing you use it in a different pr 🥳)) ? spacing is getting a little wonky looking on this page and im just trying to figure out how we can unify it a bit. right now there are specific margins on `p`'s that are first child vs last child, etc. and it seems like `gap` can simplify that

Reply:
Something like that would probably be an improvement -- things are definitely a hodgepodge of stylings and it's not clear what it'll take to make it look good.  But the instructions in the ticket were "spacings &c don't change".  I think I'll work through this again with Josh after the other changes are settled.

Anyway, I like that idea, it could simplify things.  I'll sit on it for the time being.

Reply:
So in order to make the menu highlights work better I reswizzled margins at the top level to have the space (as padding) at the *top* of each header section container.